src/config/index.js

```diff
@@ -95,6 +95,24 @@ export const config = convict({
     format: String,
     default: '',
     env: 'S3_ENDPOINT'
+  },
+  oidcJwksUri: {
+    doc: 'The URI that defines the OIDC json web key set',
+    format: String,
+    default: null,
```

Comment:
See https://github.com/mozilla/node-convict/tree/master/packages/convict#how-can-i-define-a-configuration-property-as-required-without-providing-a-default-value

Personally, I'd like to get rid of `node-convict` and align with the other repos by just using `joi`.

I was a bit surprised by

> The philosophy was to have production values be the default values

I think any config defaults are a potential footgun and tend to avoid them in favour of explicit envars. That way, there's no guessing and all .env files look consistent.

Anyway, I used null to define the config items as mandatory here, but I'm happy to default them to the `local` settings if that's what we'd prefer?

Reply:
I had to add defaults back in otherwise convicts `config.get` typed the return as `null`.
Added a `todo` to  remove the library